README.md

```diff
@@ -341,6 +342,7 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
 - [Udemy: NLP - Natural Language Processing with Python](https://www.udemy.com/nlp-natural-language-processing-with-python/)
 - [Udemy: Deep Learning: Advanced NLP and RNNs](https://www.udemy.com/deep-learning-advanced-nlp/)
 - [Udemy: Natural Language Processing and Text Mining Without Coding](https://www.udemy.com/master-natural-language-processing-nlp-text-mining-without-coding/)
+- [fast.ai 2019](https://www.fast.ai/2019/01/24/course-v3/) 
```

Comment:
This course, while amazing, is a little too broad for this repository. I'm looking for NLP-specific courses, not general AI courses that include some NLP.